File: 2018-12-23/2920/2920.c
#include <stdio.h>

int main(){
	int arr[8] = {0,};
	char *result = "";
	for(int i = 0 ; i < 8; i++)
		scanf("%d", &arr[i]);
	for(int i = 0; i < 7; i++){
		if(arr[i]+1 == arr[i+1]){
			result = "ascending";
		}else if(arr[i] == arr[i+1]+1){
			result = "descending";
		}
		else{
			result = "mixed";
			break;
		}
	}
	printf("%s", result);
}

File: 2018-12-23/2920/2920.java
import java.io.*;

class Main{
	public static void main(String[] args) throws Exception {
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in)); 
		String arr[] = bfr.readLine().split(" ");
		String result ="";
		for(int i = 0; i<7; i++){
			if(toInt(arr[i])+1 == toInt(arr[i+1]))
				result = "ascending";
			else if(toInt(arr[i]) == toInt(arr[i+1])+1)
				result = "descending";
			else {
				result = "mixed";
				break;
			}
		}
		System.out.println(result);
	}
	static int toInt(String data){
		return Integer.parseInt(data);
	}
}